BedtimeStories/tasks/VolleyBedtimeStories.1.json

```diff
@@ -0,0 +1,67 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "Task to start a Volley a bedtime story",
+    "version": "1"
+  },
+  "tags": [
+    {
+      "name": "start a volley bedtime story"
+    }
+  ],
+  "paths": {
+    "/VolleyBedtimeStories": {
+      "summary": "Volley Bedtime Stories",
+      "description": "To start a Volley bedtime story",
+      "post": {
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Input"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "When the bedtime story starts playing successfully via AudioPlayer",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/SuccessfulResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "When the given parameters fail validations - e.g. invalid requesterSkillId provided"
+          },
+          "500": {
+            "description": "When the bedtime story fails to start"
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "Input": {
+        "type": "object",
+        "properties": {
+          "requesterSkillId": {
+            "type": "string"
```

Comment:
Can you describe why you would need the requesterSkillId? We don't normally recommend having a requesterSkillId being passed around. If this is about playing a bedtime story, would the name of the bedtime story be more relevant or the preferences of a bedtime story rather than the skill that's requesting?

Reply:
We'd like to personalize the intro messages and story selections depending on which skill you are coming in from. i.e. if you are coming from our Love Taps skill we will highlight more romance-focused Bedtime Stories vs. if you are coming from Song Quiz or Yes Sire. Essentially, this is a way for us customize the experience. I have a related question, though, will this task be public by default? i.e. will other vendors be able to start using the task as soon as it is approved? If so, would be good data for us to know what other skills are trying to request a bedtime story.

Reply:
great! thank you

Reply:
Let me know if the above answer helps. Feel free to resolve the conversation, then I'll merge in the changes and ingest the schema. Please note that ingesting the task schema may take 2-3 business days.

Reply:
By the way, please also share with us your skill ID.